Source/multi.cpp

```diff
@@ -10,18 +10,18 @@ PkPlayerStruct netplr[MAX_PLRS];
 BYTE sgbPlayerTurnBitTbl[MAX_PLRS];
 char sgbPlayerLeftGameTbl[MAX_PLRS];
 int sgbSentThisCycle; // idb
-int dword_678628;     // weak
+BOOL dword_678628;    // weak

```

Comment:
If this one is bool, perhaps we should update the name from `dword_xxx` to something.

Reply:
gbShouldValidatePackage? Since it decides if multi_check_pkt_valid should be run or not

Reply:
https://github.com/diasurgical/devilution/pull/896